pydeconz/models/light/light.py

```diff
@@ -294,6 +322,17 @@ def brightness(self) -> int | None:
         """
         return self.raw["state"].get("bri")
 
+    @property
+    def supported_effects(self) -> list[LightEffect] | None:
+        """List of effects supported by a light."""
+        if (
+            "capabilities" in self.raw
+            and "color" in self.raw["capabilities"]
+            and "effects" in self.raw["capabilities"]["color"]
+        ):
+            return list(map(LightEffect, self.raw["capabilities"]["color"]["effects"]))
+        return None
```

Comment:
One final observation, I wonder if this should always return a list

Reply:
I feel `None` expresses "I don't support effects" better than an empty list ("I don't have any effects").

The HomeAssistant integration also makes the distinction - an empty list of effects shows an empty "Effects" button, while `None` won't show the "Effects" button at all.  The integration could always check if the list is empty and return `None` but, again, I feel the library differentiating between "don't support effects" and "don't have any effects" is more expressive.

Reply:
Ok, lets keep it as is